helm/linkerd2-multicluster-app/Chart.yaml

```diff
@@ -1,23 +1,23 @@
-apiVersion: "v1"
-appVersion: stable-2.11.4
+apiVersion: v2
+appVersion: stable-2.12.2
+annotations:
+  application.giantswarm.io/team: team-cabbage
 description: |
   The Linkerd-Multicluster extension contains resources to support multicluster
   linking to remote clusters
-home: https://linkerd.io
+home: https://github.com/giantswarm/linkerd2-multicluster-app
+icon: https://linkerd.io/images/logo-only-200h.png
```

Comment:
@giantswarm/team-rainbow Is there a Linkerd logo on s.giantswarm.io? Or where could we look this up? :)

Reply:
there is. Updating it.

Reply:
To answer the question, you can browse existing logos in https://github.com/giantswarm/web-assets/tree/main/assets/app-icons